Resources/Prototypes/_LostParadise/Maps/normandy.yml

```
@@ -1,76 +1,76 @@
+          mapNameTemplate: '{0} N7 Нормандия {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath: /Maps/Shuttles/DeltaV/NTES_UCLB.yml
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
+          availableJobs:
+            # Service
+            Bartender: [ 2, 2 ]
```

yamllint

[error] too many spaces inside brackets

(brackets)